WordPress/Classes/Utility/ContextManager.m

```diff
@@ -228,6 +229,18 @@ - (void)handleUIUpdate
     }
 }
 
+- (void)pruneTransactionHistory
+{
+    [self performAndSaveUsingBlock:^(NSManagedObjectContext *context) {
+        NSDate *date = [[NSCalendar currentCalendar] dateByAddingUnit:NSCalendarUnitDay
+                                                                value:-7
```

Comment:
what's the -7 value here mean?

Reply:
It's adding -7 days to the current date. So one week before whatever the current day is.